src/css.js

```diff
@@ -46,16 +44,20 @@ export const css = function(key, value) {
     for(prop in styleProps) {
       val = styleProps[prop];
       delete styleProps[prop];
-      styleProps[camelize(prop)] = val;
+      styleProps[dasherize(prop)] = val;
     }
   }
 
   each(this, element => {
     for(prop in styleProps) {
       if(styleProps[prop] || styleProps[prop] === 0) {
-        element.style[prop] = styleProps[prop];
+        const important = /!important$/.test(styleProps[prop]) ? 'important' : undefined;
+        if(typeof styleProps[prop] === 'string') {
+          styleProps[prop] = styleProps[prop].replace(/\s*!important$/, '');
+        }
```

Comment:
Sorry for being picky, how about this (one line and slightly better readable?):

```
styleProps[prop] = important ? styleProps[prop].replace(/\s*!important$/, '') : styleProps[prop];
```

Reply:
Fixed.